### DISCUSSION TITLE
Log message: OneDrive quota information is being restricted

### DISCUSSION THREAD

**Discussion by titusjan** [abraunegg/onedrive#1964] [Q&A] [answered]
I am using the OneDrive client to sync groups off our organization. For some groups I noticed the following error in the log file:

`error: onedrive quota information is being restricted. please fix by speaking to your onedrive / office 365 administrator.`

At the moment I am only downloading so it doesn't seem to have an impact, Still I would like to resolve this error somehow. 

I am an office 365 administrator of our organization, so I could make changes. The thing is, I don't know how to. The only thing related to quota that I can find is setting the site storage limits. We have this at _automatic_ for all sites, which is the default.

![automatic_storage_limits](https://user-images.githubusercontent.com/3320415/169252501-03f178c3-82a6-4041-85d5-5221fd25d0ae.png)

Can someone elaborate on this error or give me a link to a website with more info? Any help would be appreciated.

**Comment by abraunegg** [comment, marked as answer]
@titusjan 
That particular message relates to the access permissions to a given site - if you compare sites that do not generate that message, to those that do - one of your security settings / access permissions will be different - and it will not be quota related specifically.

What the restricted access permissions are doing - it is essentially not allowing the OneDrive API to read the storage details to provide a running counter as to space available online when uploading data.

In the grand scheme of things - this error is harmless if you are simply downloading data.
